pkg/autopilot/controller/signal/k0s/restart.go

```diff
@@ -1,3 +1,5 @@
+//go:build linux || darwin
```

Comment:
We don't provide macos builds, and I don't think we should be adding in conditional macos support for unit tests.

Reply:
I kept darwin support to allow running unit tests locally. Changed it to `//go:build !windows` to make it a bit cleaner.